### ISSUE TITLE
Provide ISO/DIN A4 paper size

### ISSUE THREAD

**Issue by holtwick** [lorenbrichter/ragmask#21]
The documents are provided in US Letter paper size. In Germany, but probably also in most other European countries, the DIN A4 format is most popular. It is therefore difficult to do the print out keeping the correct scale on printers and papers optimized for DIN A4.

At least for those translations of languages spoken in countries that use this paper format it should be provided in this format.

Learn more about Paper formats here: https://en.wikipedia.org/wiki/Paper_size

**Comment by holtwick** [comment]
Using this tip the scaling also works for DIN A4 printers. Probably it would be sufficient as well to put this tip on the first page of the documents: https://github.com/lorenbrichter/ragmask/blob/master/sizing/printscale.jpg

**Comment by lorenbrichter** [comment]
Yes, this is probably a good default. All pages should already be designed to fit within the common US Letter / A4 "safe area" already, but will change the PDF. Thanks!
